ministryofjustice/correspondence_tool_staff
# CT-3139 Add NCND cases into cabinet office report

## Description

This PR is to take NCND cases into "Refused fully" categories and also the exemptions related categories after being clarified from London users

## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3198
### Deployment


### Manual testing instructions

1 - Login as London users
2 - Click 'Reports" from top menu
3 - Download the Cabinet office report for FOI
4 - Check the figure
